x/ibc/core/03-connection/types/msgs.go

```diff
@@ -15,13 +15,12 @@ var _ sdk.Msg = &MsgConnectionOpenInit{}
 // NewMsgConnectionOpenInit creates a new MsgConnectionOpenInit instance
 //nolint:interfacer
 func NewMsgConnectionOpenInit(
-	connectionID, clientID, counterpartyConnectionID,
+	clientID,
 	counterpartyClientID string, counterpartyPrefix commitmenttypes.MerklePrefix,
 	version *Version, signer sdk.AccAddress,
 ) *MsgConnectionOpenInit {
-	counterparty := NewCounterparty(counterpartyClientID, counterpartyConnectionID, counterpartyPrefix)
+	counterparty := NewCounterparty(counterpartyClientID, "", counterpartyPrefix)
```

Comment:
nit can we update the comment to state that the counterparty connectionID is empty?